score.py

```diff
@@ -0,0 +1,214 @@
+from collections import Counter, namedtuple
+
+from nameinternal import get_card_metadata, get_score_bonus
+from save import Savefile
+
+class Score:
+    def __init__(self, name: str, count: int, score: int = 0, should_show: bool = False):
```

Comment:
If we can call `Score` with no arguments, then both `name` and `count` need to have pre-defined values.